site/resources.md

```diff
@@ -70,9 +70,9 @@ Resources
 
   _add description_
 
-* MetPy (basic calculations)
+* [**MetPy (basic calculations)**](https://nbviewer.jupyter.org/github/Unidata/pyaos-ams-2021/blob/master/notebooks/analysis/metpy_basics.ipynb)
 
```

Comment:
Thanks for bolding the title.
Now for the proper spacing between the title and description, remove the new line, and add two spaces at the end of the title link.  If you pull from the master version first, you'll notice a new comment at the top of this file, explaining that the spaces are necessary for markdown to interpret and render the webpage properly.

Reply:
Ok, try that.

Reply:
Great!  Thanks.